packages/expo-contacts/src/Contacts.ts

```diff
@@ -325,28 +323,29 @@ export async function removeContactAsync(contactId: string): Promise<any> {
   return await ExpoContacts.removeContactAsync(contactId);
 }
 
-export async function writeContactToFileAsync(contactQuery: ContactQuery = {}): Promise<?string> {
+export async function writeContactToFileAsync(
+  contactQuery: ContactQuery = {}
+): Promise<string | undefined> {
   if (!ExpoContacts.writeContactToFileAsync) {
     throw new UnavailabilityError('Contacts', 'writeContactToFileAsync');
   }
   return await ExpoContacts.writeContactToFileAsync(contactQuery);
 }
 
-// TODO: Evan: Test
 export async function presentFormAsync(
-  contactId: ?string,
-  contact: ?Contact,
+  contactId?: string,
```

Comment:
```suggestion
  contactId?: string | null,
```

From the caller's perspective, passing in `null` is more common than needing to pass in `undefined` specifically. The `?` will make it so that people can call `presentFormAsync()` with no arguments and implicitly the type of `contactId` will be `string | null | undefined`, which we handle fine.